core/protocols/protocol-core/src/main/java/software/amazon/awssdk/protocols/core/OperationInfo.java

```diff
@@ -133,7 +133,7 @@ public boolean hasEvent() {
      * @param <T> Type of metadata being requested.
      * @return The value of the additional metadata being requested or null if it's not present.
      */
-    public <T> T addtionalMetadata(OperationMetadataAttribute<T> key) {
+    public <T> T additionalMetadata(OperationMetadataAttribute<T> key) {
```

Comment:
Unfortunately since this is a protected API, we shouldn't fix the spelling without deprecating the 'wrong' method name and creating a new one. We support people using newer core library versions with older client versions, and this could cause them to break by updating a core library.

We might be able to get away with fixing this when we do a minor version bump, but we'd probably want to lump it in with other changes to justify the annoyance to customers.

Reply:
Undoing change.